docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,53 @@ discretize
 DiscretizationMethod
 FIST
 ```
+# FIST  
+Examples for the FIST algorithm. 
+
+### Usage  
+  
+```@example overview
+# Import required Libraries
+using GeoStats
```

Comment:
Why is GeoStats imported? We don't need it here, and it errors in the Documenter run on CI since it is not part of the project.

Reply:
Fixing

Reply:
@serenity4 The plots are not getting calculated/being displayed if I don't import GeoStats. How should I approach this problem?

Reply:
![Screenshot from 2021-03-25 01-52-06](https://user-images.githubusercontent.com/14348863/112378268-ba521e80-8d0c-11eb-8637-d0d958b33055.png)

Reply:
Hmm I have no idea, @juliohm? Why would that import be related in any way?

Reply:
I also have no idea what is happening. GeoStats.jl shouldn't be at all related to the plots in Meshes.jl @atreyamaj could you please elaborate on how you're executing these scripts? Are you following the steps I shared with you on Zulip for building documentation?

One thing you may need to do is add Plots.jl to the `docs/Project.toml` environment *using the Pkg manager*:

```julia
cd(".julia/dev/Meshes/docs")
] add Plots
```

Reply:
I'll try this and get back to you!

Reply:
@juliohm  This didn't work.